hivemq-edge/src/frontend/src/modules/Workspace/components/edges/MonitoringEdge.tsx

```diff
@@ -55,6 +55,7 @@ const MonitoringEdge: FC<EdgeProps> = (props) => {
           <ObservabilityEdgeCTA source={source} style={style} onClick={handleOpenObservability} />
           {policyRoutes && (
             <DataPolicyEdgeCTA
+              data-testid="policy-panel-trigger"
```

Comment:
Is there a specific reason for this `prop`, it is not used on the component itself, and the `data-testid` is not unique

Reply:
Thanks @tolumq, `data-testid` is the convention for identifying components for testing purposes only (cypress, check for the `cy.getByTestId` command). It's not unique but being component-tested, they will always be unique on the "page" @tolumq